Makefile

```diff
@@ -26,4 +26,8 @@ lint:
 
 # How we install ktlint in the team. 
 lint-install:
-	./scripts/get-ktlint.sh 
\ No newline at end of file
+	./scripts/get-ktlint.sh
+
+# Run kotlin binary validator
+run-binary-validator:
```

Comment:
technically, this script runs *and updates* the .api files. I think this needs to be updated to be more descriptive of what it's doing.